resources/views/store/index.blade.php

```diff
@@ -18,7 +18,7 @@
 @extends('master')
 
 @section('content')
-    @include("store.header")
+    @include('store.header')
```

Comment:
I'd rather we not make these kind of changes in pull requests that have nothing to do with them

Reply:
there's just no place to do this because I don't think I'll do full cleanup on this thing and having pr with just this is dumb.